openai/version.py

```diff
@@ -1 +1 @@
-VERSION = "0.10.1"
+VERSION = "0.10.5"
```

Comment:
why is the version bump .5 instead of .2?

Reply:
fixed